workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -245,6 +230,36 @@ private static void Generate()
             }
         }
 
+        private static string GetDefaultLoggerOutputPath(GdkToolsConfiguration toolsConfig = null)
+        {
+            toolsConfig = toolsConfig ?? GdkToolsConfiguration.GetOrCreateInstance();
+            return Path.GetFullPath(Path.Combine(toolsConfig.CodegenLogOutputDir, "codegen-output.log"));
+        }
+
+        private static string GetSchemaCompilerPath()
+        {
+            var schemaCompilerPath = SchemaCompilerPath;
+
+            switch (Application.platform)
+            {
+                case RuntimePlatform.WindowsEditor:
+                    schemaCompilerPath = Path.ChangeExtension(schemaCompilerPath, ".exe");
+                    break;
+                case RuntimePlatform.LinuxEditor:
```

Comment:
Pretty sure that Linux is unsupported too? We don't download the `schema_compiler` for that platform